sharing/standard.go

```diff
@@ -56,9 +61,62 @@ func (handler *Handler) handleStandard(ctx context.Context, w http.ResponseWrite
 	}
 
 	pr.access = access
+
+	q := r.URL.Query()
+
+	// flag used for get locations request.
+	locationsFlag := queryFlagLookup(q, "locations", false)
+	if locationsFlag {
+		err = handler.serveLocations(ctx, w, pr)
+		if err != nil {
```

Comment:
You can use `return handler.serveLocations(ctx, w, pr)` directly.

Reply:
yes. You're right